### DISCUSSION TITLE
Error when loading and saving?

### DISCUSSION THREAD

**Discussion by pwylcrow** [SciSharp/TensorFlow.NET#1009] [Q&A]
Do you have an example using the keras api of loading and saving a model? Currently when I save a model using model.save it works fine but loading it gives me a null exception error with no more info than that. (Yes the path exists and the files are there).

**Comment by SanftMonster** [comment]
Could you please tell us the version you used? And it's better if an minimal code to reproduce it is avaliable.

**Reply by pwylcrow** [reply]
I resolved the issue. I can only load h5 weights but not full blown models. I resolved it by recreating an empty model then loading the weights.

**Reply by SanftMonster** [reply]
Glad to hear that :). Would you mind giving us a minimal example to reproduce the error you met before? It could help us to complete this feature and resolve this possible error for other users. Thank you!

**Reply by pwylcrow** [reply]
Sure, just try to save your model with a model.save and specify the file with a .pb extension.

**Reply by SanftMonster** [reply]
Could you please tell more things about "specify the file with a .pb extension"? I didn't understand it 😀
